src/engraving/dom/select.cpp

```diff
@@ -552,6 +558,25 @@ void Selection::appendTupletHierarchy(Tuplet* innermostTuplet)
     }
 }
 
+void Selection::appendGuitarBend(GuitarBend* guitarBend)
+{
+    if (!guitarBend) {
+        return;
+    }
+
+    m_el.push_back(guitarBend);
+
+    if (GuitarBendHold* hold = guitarBend->holdLine()) {
```

Comment:
This seems to cause a tiny inconsistency: most spanners are selected if and only if they are entirely contained in the selected range, but a bend hold line is selected if and only if the start is contained in the selected range. In particular, this looks not ideal when the hold line extends further to the right than the selected range.